### PR TITLE
[New Feature] Add Links to StackOverflow and Joomla@StackExchange

### PR DESCRIPTION
**Pull request by zero-24**
Pull request for issue #169 

Beside our forum and Facebook pages, we have a lot of other places, where people contribute to Joomla. Most recognised among them are StackOverflow (stackexchange.com) and Joomla Q&A (joomla.stackexchange.com).

There could be a button like the one for Docs that will link to his/her profile on SO/JSE:

```
https://stackexchange.com/users/____
https://joomla.stackexchange.com/users/____
```
where `____` will be filled out by the user with his/her ID on the respective site.

These fields could be optional and the icon won't appear if the user doesn't fill out the fields with the link to his/her profile.

This is a way to recognise volunteers on these important channels.

### CODE REVIEW

File: database/installation/volunteers.joomla.org.sql [hunk 1]
```diff
@@ -2521,6 +2521,8 @@ CREATE TABLE `vol_volunteers_volunteers` (
   `website` varchar(255) NOT NULL,
   `github` varchar(255) NOT NULL,
   `certification` varchar(255) NOT NULL,
+  `stackexchange` VARCHAR(255) NOT NULL,
+  `joomlastackexchange` VARCHAR(255) NOT NULL,
```
Comment (nibra) [inline database/installation/volunteers.joomla.org.sql:2525]: The ids are actually integers....

**Comment by zero-24** [comment]
I did have to revert the last commit as the actual URL looks like this: https://stackexchange.com/users/2018/andyp so we can't save that into a integer ;)

**Comment by nibra** [comment]
Of course - it's the URL, not the id. Sorry for the irritation!

**Comment by sanderpotjer** [comment]
@zero-24 thanks for the PR, appreciated! Should be live soon

**Comment by zero-24** [comment]
Thanks

**Pull request merged**